### PR TITLE
Fix minio.yaml wrong secret key.

### PR DESCRIPTION
**Pull request by EDGsheryl**
This wrong key caused:
```
MountVolume.SetUp failed for volume "mlpipeline-minio-artifact" : references non-existent secret key: accesskey
```

Your version
image

### CODE REVIEW

File: proposed/kubeflow/v0.6.3/templates/istio.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+apiVersion: v1
+data:
+  clusterRbacConfig: {{ .Values.istio.clusterRbacConfig | quote }}
+kind: ConfigMap
```
Comment (EDGsheryl) [inline proposed/kubeflow/v0.6.3/templates/istio.yaml]: I have fix this duplicated.

File: proposed/kubeflow/v0.6.3/templates/minio.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+kind: ConfigMap
+metadata:
+  labels:
+    app: minio
+  name: pipeline-minio-parameters
+  namespace: kubeflow
+---
+apiVersion: v1
+data:
+  accesskey: {{ .Values.minio.accessKey | b64enc | quote }}
```
Comment (EDGsheryl) [inline proposed/kubeflow/v0.6.3/templates/minio.yaml]: accesskey should be lowercase

File: proposed/kubeflow/v0.6.3/templates/minio.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    spec:
+      containers:
+      - args:
+        - server
+        - /data
+        env:
+        - name: MINIO_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: accesskey
```
Comment (EDGsheryl) [inline proposed/kubeflow/v0.6.3/templates/minio.yaml]: accesskey should be lowercase

File: proposed/kubeflow/v0.6.3/templates/minio.yaml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+              name: mlpipeline-minio-artifact
+        - name: MINIO_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              key: secretkey
```
Comment (EDGsheryl) [inline proposed/kubeflow/v0.6.3/templates/minio.yaml]: lowercase here too

**Comment by PennyScissors** [comment]
Hello @EDGsheryl, I'm working on cleaning up this repository, and noticed we lost track of this PR. Should I go ahead and close this, or is it still relevant? You can always submit a new one if you like.

**Comment by PennyScissors** [comment]
Closing this as there hasn't been any activity

**Pull request closed without merging**